pom.xml

```diff
@@ -659,9 +659,14 @@
         <artifactId>jackson-dataformat-smile</artifactId>
         <version>${jackson.version}</version>
       </dependency>
+      <dependency>
+        <groupId>com.fasterxml.jackson.dataformat</groupId>
+        <artifactId>jackson-dataformat-yaml</artifactId>
+        <version>${jackson.version}</version>
+      </dependency>
       <dependency>
         <groupId>com.fasterxml.jackson.jaxrs</groupId>
-        <artifactId>jackson-jaxrs-json-provider</artifactId>
+        <artifactId>jackson-jaxrs-base</artifactId>
```

Comment:
Instead of calling out all of the Jackson dependencies separately in the top level pom, there is a Jackson BOM that should help with this.

https://github.com/FasterXML/jackson-bom

specifically add to the dependency management section in the top level pom:

```
        <dependency>
            <groupId>com.fasterxml.jackson</groupId>
            <artifactId>jackson-bom</artifactId>
            <version>${jackson.version}</version>
            <scope>import</scope>
            <type>pom</type>
        </dependency>  
```

This will ensure all the Jackson dependencies are the right version. This should avoid even having to declare all the individual jackson dependencies in the top level pom w/ versions. 

PS - you can do the same with Spring and Netty. There are a few other projects that produce BOM artifacts to specifically help with this .